build-ci/jenkins.rst

```diff
@@ -0,0 +1,130 @@
+##############################
+Jenkins ci.lsst.codes Overview
+##############################
+
+Data Management operates a Jenkins continuous integration cluster at https://ci.lsst.codes.
+You will typically use Jenkins to run the ``stack-os-matrix`` job to build and run unit tests in the LSST Stack as part of `DM's development workflow <processes/workflow.rst>`_.
+See :doc:`jenkins-stack-os-matrix` for details.
+
+Logging in
+==========
+
+Log into Jenkins by visiting https://ci.lsst.codes and authenticate with your GitHub credentials.
+Any member of the `lsst`_, `lsst-dm`_, or `lsst-sqre`_ GitHub organizations is authorized to use https://ci.lsst.codes.
+See the :doc:`../getting-started/onboarding` for more information.
+
+Using the Jenkins dashboard
+===========================
+
+This section summarizes how to use the Jenkins dashboard, which is the first view you see when browsing to https://ci.lsst.codes.
+`Jenkins's own dashboard documentation <https://jenkins.io/doc/book/blueocean/dashboard/>`_ provides additional information.
+
+Browsing Jenkins jobs
+---------------------
+
+From https://ci.lsst.codes, you see a listing of Jenkins jobs.
+Each job fulfills a different purpose, see :ref:`jenkins-jobs`.
+
+To access a job, click on the job's name.
+This takes you to a page where you can run the job and review past runs.
+
+Favoriting Jenkins jobs
+-----------------------
+
+From the https://ci.lsst.codes dashboard, you can click on the star icon next to any job.
+This will favorite the Job.
+Favorited jobs appear at the top of the homepage, before other jobs.
+
+We recommend that you favorite the ``stack-os-matrix`` job.
+
+Searching for Jenkins jobs
+--------------------------
+
+From the https://ci.lsst.codes dashboard, you can filter the listing of jobs by using the search box at the top of the page.
+
+Jenkins jobs are organized in directories, so it's helpful to search by directory prefix to find the right job.
+The directories are:
+
+- ``qserv``
+- ``release``
+- ``science-pipelines``
+- ``sims``
+- ``sqre``
+
+Understanding the weather icons
+-------------------------------
+
+Jenkins represents a job's historical patterns of run successes and failures with weather icons throughout the dashboard interface.
+
+============= ==============================
+Icon          Percent of recent runs passing
+============= ==============================
+Sunny         >80%
```

Comment:
Is it worth actually putting the icons people will see?  If it's hard to extract the icons, I think the words are understandable.